reactive/cuda.sh

```diff
@@ -235,8 +235,8 @@ function all::x86_64::install_cuda() {
     [ -f ${INSTALL_PKG} ] && rm -f ${INSTALL_PKG}
     wget ${ROOT_URL}/${UBUNTU_VERSION}/x86_64/${INSTALL_PKG}
     dpkg -i /tmp/${INSTALL_PKG}
-    apt update && \
-    apt install -yqq --allow-downgrades --allow-remove-essential --allow-change-held-packages \
+    apt-get update -qq && \
+    apt-get install -yqq --allow-downgrades --allow-remove-essential --allow-change-held-packages --no-install-recommends \
```

Comment:
I'd recommend `apt` over `apt-get` as `apt-get` is lower level interations, but if logging is messed up `apt-get` is fine.